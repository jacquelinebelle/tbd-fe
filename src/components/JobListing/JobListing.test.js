import React from "react";
import { shallow } from "enzyme";
import { JobListing } from "./index";

const mockJob = {
  lastModified: "01/26/2020",
  title: "FrontEnd Engineer",
  link: "randomlink.com",
  location: "Dallas, TX",
  salary: "400000",
  company: "Tronkat Inc.",
  snippet: "This is a random snippet",
  type: "Full Time"
};

describe("Job Listing", () => {
  let wrapper;

  beforeEach(function() {
    wrapper = shallow(<JobListing {...mockJob} />);
  });

  it("should match snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
