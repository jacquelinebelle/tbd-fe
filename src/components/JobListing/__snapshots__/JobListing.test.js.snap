// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Job Listing should match snapshot 1`] = `
<article
  className="job"
>
  <p
    className="job-date"
  >
    Last modified: 
    01/26/2020
  </p>
  <h3
    className="job-title"
  >
    <a
      className="job-title"
      href="randomlink.com"
    >
      FrontEnd Engineer
    </a>
  </h3>
  <h5
    className="job-location"
  >
    Dallas, TX
  </h5>
  <h5
    className="job-salary"
  >
    Full Time
    : 
    400000
  </h5>
  <p
    className="job-company"
  >
    Tronkat Inc.
  </p>
  <p
    className="job-description"
  >
    This is a random snippet
  </p>
</article>
`;
